src/services/codefixes/changeExtendsToImplementsFix.ts

```diff
@@ -0,0 +1,33 @@
+/* @internal */
+namespace ts.codefix {
+    registerCodeFix({
+        errorCodes: [Diagnostics.Cannot_extend_an_interface_0_Did_you_mean_implements.code],
+        getCodeActions: (context: CodeFixContext) => {
+            const sourceFile = context.sourceFile;
+            const start = context.span.start;
+            const token = getTokenAtPosition(sourceFile, start);
+            const textChanges: TextChange[] = [];
+
+            if (token.kind === SyntaxKind.Identifier && token.parent.parent.kind === SyntaxKind.HeritageClause) {
+                const children = (<HeritageClause>token.parent.parent).getChildren();
+                ts.forEach(children, child => {
+                    if (child.kind === SyntaxKind.ExtendsKeyword) {

```

Comment:
Fixed. The edit is to change extends -> implements, and if implements is already present, change implements -> ','